src/http-driver.js

```diff
@@ -103,14 +108,12 @@ function makeHTTPDriver({eager = false} = {eager: false}) {
       .map(reqOptions => {
         let response$ = createResponse$(reqOptions)
         if (eager || reqOptions.eager) {
-          response$ = response$.replay(null, 1)
-          response$.connect()
+          response$ = response$.publishReplay(null, 1).connect()
+          return response$
         }
         response$.request = reqOptions
         return response$
-      })
-      .replay(null, 1)
-    response$$.connect()
+      }).shareReplay(null, 1)
```

Comment:
So to answer the question, try finding an initial value that works. For `response$$`, the initial value could be `Observable.empty()`, meaning: if someone subscribes to `response$$` before anything happened on that, it will receive an empty Observable, because "Behaviors" always emit something. They are "values over time".